tests/test_authenticated_client.py

```diff
@@ -23,6 +23,7 @@ def test_place_order_input_2(self, dc):
             r = dc.place_order('BTC-USD', 'buy', 'limit',
                                cancel_after='123', tif='ABC')
```

Comment:
the tif param needs to be renamed, 'time-in-force'

Reply:
Ahh yep just seen #317